definitions/scenarios/self_upgrade.rb

```diff
@@ -67,11 +67,8 @@ class SelfUpgrade < SelfUpgradeBase
     def downstream_self_upgrade(pkgs_to_update)
       ForemanMaintain.enable_maintenance_module
 
-      dnf_options = req_repos_to_update_pkgs.map do |id|
-        "--enablerepo=#{id}"
-      end
       add_step(Procedures::Packages::Update.new(packages: pkgs_to_update, assumeyes: true,
-        dnf_options: dnf_options))
```

Comment:
This should answer your question @evgeni -- we removed this option and so this is broken entirely :)

Reply:
Oooh, so this should have a

Fixes: 9ac8a20cda789c0d733396f0c92bf9b45d25901c